bindings/r/tests/testthat/_snaps/delimiter.md

```diff
@@ -110,24 +114,19 @@
       node_children_print(node)
     Output
       S-Expression
-      (subset2 [(1, 0), (1, 5)]
-        function: (identifier [(1, 0), (1, 1)])
-        arguments: (arguments [(1, 1), (1, 5)]
-          "[[" [(1, 1), (1, 3)]
-          argument: (argument [(1, 3), (1, 4)]
-            value: (float [(1, 3), (1, 4)])
-          )
-          argument: (argument [(1, 4), (1, 5)]
-            value: (unmatched_delimiter [(1, 4), (1, 5)]
-              "]" [(1, 4), (1, 5)]
-            )
-          )
-          "]]" [(1, 5), (1, 5)]
-        )
+      (identifier [(1, 0), (1, 1)])
+      
+      Text
+      x
+      
+      S-Expression
+      (ERROR [(1, 1), (1, 5)]
+        "[[" [(1, 1), (1, 3)]
+        (ERROR [(1, 4), (1, 5)])
       )
```

Comment:
With `x[[2]`, I'm a little surprised that it couldn't detect that we are in a `subset2` node, but I guess the `]` really throws things off.

Nevertheless the new error still allows us to do

```
x[[2]
 ~~~~
 ^
 |
 - Syntax error
```